public/app/features/alerting/unified/components/receivers/TemplateDataDocs.tsx

```diff
@@ -0,0 +1,204 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data/src';
+import { Stack, useStyles2 } from '@grafana/ui/src';
+
+export function TemplateDataDocs() {
+  return (
+    <Stack gap={2}>
+      <TemplateDataTable caption="Template Data" dataItems={GlobalTemplateData} />
+      <TemplateDataTable
+        caption="Alert template data - available only when in the context of an Alert"
+        dataItems={AlertTemplateData}
+      />
+    </Stack>
+  );
+}
+
+function TemplateDataTable({ dataItems, caption }: { dataItems: TemplateDataItem[]; caption: string }) {
+  const styles = useStyles2(getTemplateDataDocsStyles);
+
+  return (
+    <table className={styles.table}>
+      <caption>{caption}</caption>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Type</th>
+          <th>Notes</th>
+        </tr>
+      </thead>
+      <tbody>
+        {dataItems.map(({ name, type, notes }, index) => (
+          <tr key={index}>
+            <td>{name}</td>
+            <td>{type}</td>
+            <td>{notes}</td>
+          </tr>
+        ))}
+      </tbody>
+    </table>
+  );
+}
+
+const getTemplateDataDocsStyles = (theme: GrafanaTheme2) => ({
+  table: css`
+    table-layout: fixed;
+    border-collapse: collapse;
+    width: 100%;
+
+    caption {
+      caption-side: top;
+      color: ${theme.colors.text.primary};
+      font-size: ${theme.typography.h4.fontSize};
+    }
+
+    td,
+    th {
+      padding: ${theme.spacing(1, 2)};
+    }
+
+    thead {
+      font-weight: ${theme.typography.fontWeightBold};
+    }
+
+    thead th:nth-child(1) {
+      width: 180px;
+    }
+
+    thead th:nth-child(2) {
+      width: 120px;
+    }
+
+    thead th:nth-child(3) {
+      width: auto;
+      max-width: 400px;
+    }
+
+    tbody tr:nth-child(2n + 1) {
+      background-color: ${theme.colors.background.secondary};
+    }
+
+    tbody td:nth-child(1) {
+      font-weight: ${theme.typography.fontWeightBold};
+    }
+
+    tbody td:nth-child(2) {
+      font-style: italic;
+    }
+  `,
+});
+
+interface TemplateDataItem {
+  name: string;
+  type: 'string' | '[]Alert' | 'KeyValue' | 'time.Time';
+  notes: string;
+}
+
+const GlobalTemplateData: TemplateDataItem[] = [
+  {
+    name: 'Receiver',
+    type: 'string',
+    notes: 'Name of the contact point that the notification is being sent to.',
+  },
+  {
+    name: 'Status',
+    type: 'string',
+    notes: 'firing if at least one alert is firing, otherwise resolved',
```

Comment:
Perhaps this could be clearer?  "Fires if at least one alert (rule) is firing." "Otherwise resolved" - not sure what the sense is here?

Reply:
I copied all of the descriptions here just from our docs :) I'm happy to improve that in both places 
https://grafana.com/docs/grafana/latest/alerting/contact-points/message-templating/template-data/